tests/system_tests/test_plans_system.py

```diff
@@ -0,0 +1,14 @@
+from bluesky import RunEngine
+from dodal.beamlines.b01_1 import oav, sample_det, sample_stage
+
+from test_rig_bluesky.plans import snapshot
+
+
+def test_snapshot():
```

Comment:
Should we run these via blueapi or standalone? Standalone is a more isolated test but a blueapi REST call is more end-to-end

Reply:
It might be good to swap to blueapi only for the system tests in future, but currently it is broken due to the WIP auth changes, so I can't confirm this works with it.